src/components/Inputs/ImportStrategy.js

```diff
@@ -0,0 +1,29 @@
+import i18n from '@dhis2/d2-i18n'
+import React from 'react'
+import { RadioGroup } from '../FinalFormComponents/RadioGroup'
+
+export const OPTION_CREATE_AND_UPDATES = {
+    value: 'CREATE_AND_UPDATES',
```

Comment:
This is `CREATE_AND_UPDATE` in the version on play

Reply:
Fixed